config/default.js

```diff
@@ -21,6 +21,9 @@ module.exports = {
       revisions: 'revisions',
       save: 'save',
     },
+    plugins: {
```

Comment:
Talking with Ayush:

1. check if `content.plugins.directory` exists
-no? make configPlugins.search a blank array
2. if exists, is it an array?
-yes? send to plugabilly
-no? do 3.
3. if not array, is it a string?
-yes? put in array, send to pb
-no? error out?

@ayush2k sound right?
@Snugug sound legal?